.github/workflows/iroha2-pr.yml

```diff
@@ -13,22 +13,45 @@ env:
 jobs:
   long-tests:
     runs-on: [ self-hosted, Linux ] #ubuntu-latest
+    container: rust:1.52-buster
     steps:
       - uses: actions/checkout@v2
       - uses: actions/cache@v2
         with:
           path: ${{ env.CACHE_PATHS }}
           key: ${{ hashFiles('Cargo.toml') }}
+      - name: Install dependencies
+        run: |
+          apt-get update
+          apt-get install -y --no-install-recommends \
+            build-essential \
+            ca-certificates \
+            clang \
+            llvm-dev \
+            libssl-dev
       - name: Run long tests
         run: cargo test --workspace --verbose -- --ignored --test-threads=1 long
   test-docker:
-    runs-on: [ self-hosted, Linux ] #ubuntu-latest
+    runs-on: ubuntu-latest
```

Comment:
Why not self hosted?

Reply:
Because there are some weird issues with docker there. Im not sure about the setup, so i just left docker tests on github runner.

It just didn't worked out of box, as i remember.

Reply:
Let's make an issue then and ask @kuvaldini to fix it if he knows how

Reply:
How can i help?